README.md

```diff
@@ -18,50 +18,82 @@
 ## Requirements
 
 * Mac OS X or Linux
-* Java 11, 64-bit
+* Java 11.0.8 or later, 64-bit
 * Python 2.6+ (for running with the launcher script)
-* Docker (for building and running tests)
+* Docker (for building and running tests, and for building documentation)
```

Comment:
The "for building" covers building documentation

Reply:
We discussed and agreed to remove the parentheses and just say that Docker is a hard requirement (as various parts of the build use it and we expect more in the future)